src/components/vm/disks/diskEdit.jsx

```diff
@@ -90,7 +90,7 @@ const BusRow = ({ onValueChanged, dialogValues, diskDevice, idPrefix, shutoff, s
         <FormGroup fieldId={`${idPrefix}-bus-type`} label={_("Bus")}
                    labelIcon={!shutoff &&
                        <Popover bodyContent={_("Machine must be shut off before changing bus type")}>
-                           <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                           <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-092041-0b751cdc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)